sequencer/src/bin/deploy.rs

```diff
@@ -295,11 +295,15 @@ async fn deploy_light_client_contract<M: Middleware + 'static>(
         )
         .await?;
 
-    // Link with LightClient's bytecode artifacts
-    let bytecode_path = Path::new(env!("CARGO_MANIFEST_DIR"))
-        .join("../contracts/out/LightClient.sol/LightClient.json");
-    let json: Value = serde_json::from_reader(BufReader::new(File::open(bytecode_path)?))?;
-
+    // Link with LightClient's bytecode artifacts. We include the unlinked bytecode for the contract
+    // in this binary so that the contract artifacts do not have to be distributed with the binary.
+    // This should be fine because if the bindings we are importing are up to date, so should be the
```

Comment:
I think currently we don't always have the contract artifacts when we run `cargo test` etc. because everything that cargo test needs is in the bindings. If we want this to work we either need to check in the artifacts or always compile the contracts before we run tests.

Reply:
Didn't see your comment.

Reply:
Yeah, I ended up checking in the artifacts (just the one that is needed for the build, for now). My other idea, loading the artifacts at runtime and making sure they got copied into the docker image, ended up being a bigger headache because there's like 5 different places where we build docker images (we should fix that eventually)